docs/source/examples/dcgan.rst

```diff
@@ -0,0 +1,358 @@
+DCGAN: Generate the images with Deep Convolutional GAN
+******************************************************
+
+.. currentmodule:: chainer
+
+0. Introduction
+================
+
+In this tutorial, we generate images with **generative adversarial network (GAN)**.
+It is a kind of generative model with deep neural network, and often applied to
+the image generation. The GAN technique is also applied to 
+`PaintsChainer <https://paintschainer.preferred.tech/index_en.html>`_,
+a famous automatic coloring service.
+
+.. image:: ../../image/dcgan/generated-images.gif
+
+In the tutorial, you will learn the following things:
+
+1. Generative Adversarial Networks (GAN)
+2. Implementation of DCGAN in Chainer
+
+1. Generarive Adversarial Networks (GAN)
+=========================================
+
+1.1 What is the GAN?
+---------------------
+
+As explained in the GAN tutorial in NIPS 2016 [1], the generative models can be classified into the categories as shown in the following figure:
+
+.. figure:: ../../image/dcgan/class-generative-model.png
```

Comment:
@keisuke-umezawa This file seems not to exist. Could you check if this path is correct?

Reply:
@mitmul Sorry, I added the image.